### PR TITLE
PDA: You can now select ringtones from a preset list

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Allows you to either insert manually a ringtone or to select it from a dropdown list.
## Why It's Good For The Game

We currently have no in-game way for players to now what ringtones are available without code diving, this PR solves this issue.
## Images of changes

PR11

## Testing

- Did set the ringtone manually and using the preset list
## Changelog
:cl:
tweak: PDA: You can now select ringtones from a preset list
/:cl:

### CODE REVIEW

**Comment by spacegeese** [comment]
imagine a secoff using this strat to find tots

**Comment by Henri215** [comment]
> imagine a secoff using this strat to find tots

What strat are you talking about?

**Comment by TORE01** [comment]
> imagine a secoff using this strat to find tots

The secret code you push in to access the uplink is probably not going to be listed.

**Comment by Henri215** [comment]
> > imagine a secoff using this strat to find tots
> 
> The secret code you push in to access the uplink is probably not going to be listed.

Yeah, its not listed.

File: tgui/packages/tgui/interfaces/pda/pda_messenger.js [hunk 1]
```diff
@@ -188,16 +189,24 @@ export const MessengerList = (props, context) => {
...
+          <Button icon="bell" onClick={() => act('Ringtone')}>
+            Set Custom Ringtone
+          </Button>
+          <Button>
+          <Dropdown
+            selected={ringtone}
+            width="110px"
+            options={Object.keys(ringtone_list)}
+            onSelected={(value) => act('Available_Ringtones', {selected_ringtone: value,})}
+          />
```
Comment (Sirryan2002) [inline tgui/packages/tgui/interfaces/pda/pda_messenger.js:208]:
```suggestion
            <Dropdown
              selected={ringtone}
              width="110px"
              options={Object.keys(ringtone_list)}
              onSelected={(value) => act('Available_Ringtones', {selected_ringtone: value,})}
            />
```

**Review by Sirryan2002 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]
deconflict for merge

**Pull request merged**